cmd/rev-upgrade/main.go

```diff
@@ -0,0 +1,187 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sync"
+
+	"github.com/BurntSushi/toml"
+	git "gopkg.in/src-d/go-git.v4"
+)
+
+type project struct {
+	Name     string
+	Revision string
+}
+
+type projects struct {
+	Projects []project
+}
+
+func init() {
+	flag.Usage = func() {
+		fmt.Println("\ngo run ./cmd/rev-upgrade/main.go -p \"project name\" [-r \"revision\"]")
+		flag.PrintDefaults()
+	}
+}
+
+func main() {
+	var (
+		prj    string
+		newRev string
+		oldRev string
+
+		w   *git.Worktree
+		err error
+	)
+
+	flag.StringVar(&prj, "p", "", "project name (e.g.: gopkg.in/src-d/go-mysql-server.v0)")
```

Comment:
maybe we could make this optional and make `gopkg.in/src-d/go-mysql-server.v0` the default, since it's the main (and only for now) use of the script